util/src/main/java/io/grpc/util/AdvancedTlsX509TrustManager.java

```diff
@@ -356,6 +375,17 @@ void verifyPeerCertificate(X509Certificate[] peerCertChain, String authType, SSL
         throws CertificateException;
   }
 
+  /**
+   * Builds a new {@link AdvancedTlsX509TrustManager}. By default, no trust certificates are loaded
+   * after the build. To load them, use one of the following methods: {@link
+   * AdvancedTlsX509TrustManager#updateTrustCredentials(X509Certificate[])}, {@link
+   * AdvancedTlsX509TrustManager#updateTrustCredentialsFromFile(File, long, TimeUnit,
+   * ScheduledExecutorService)}, {@link AdvancedTlsX509TrustManager#updateTrustCredentialsFromFile
+   * (File, long, TimeUnit, ScheduledExecutorService)}. By default, {@link
```

Comment:
The default and behavior when called seem they should be placed on the set methods.

Reply:
Moved 'Verification' and 'SslSocketAndEnginePeerVerifier' comments to the respected setters